modules/slugs/includes/qtranslate-slug-admin.php

```diff
@@ -30,6 +30,11 @@
 function qts_taxonomies_hooks() {
     global $qtranslate_slug;
 
+    $add_slugs_hooks=apply_filters( 'qts_add_tax_slugs_hook', array() );
+    foreach ( $add_slugs_hooks as $hook ){
+        add_action( $hook, 'qts_show_add_term_fields' );
+    }
+
```

Comment:
The filter/action are still there.

Reply:
fixed, 47e425f76a21f14b8818d34425e19c6951964b91